int32 isPowered
int32 isCalibrated
int32 workingMode
int32 esStatus
int32 moniteurSpeed
int32 shoulder
int32 elbow
int32 wrist
int32 isSettled
int32 flag
